app/features/hit_fox_feed.rb

```diff
@@ -0,0 +1,33 @@
+require 'github_feed'
+require 'yajl'
+
+class HitFoxFeed
+  def initialize(user, org, token)
+    @feed = GithubFeed.new(user,org, token)
```

Comment:
Add space after first comma or remove the space after the second one. Either way we should keep things consistent